rpm_spec/subpackages/manageiq-gemset

```diff
@@ -12,7 +12,7 @@ Requires: libxslt
 Requires: nfs-utils
 Requires: openscap-scanner
 Requires: openssl
-Requires: nodejs
+BuildRequires: nodejs
```

Comment:
Minor, but can you move this up to line 3 or 4 to be alongside the other BuildRequires ?

Reply:
Sure, I must have moved it down when resolving the conflict.